test/integration/objectid.test.js

```diff
@@ -18,13 +19,15 @@ describe('ObjectId', function () {
     test: function (done) {
       var configuration = this.configuration;
       var client = configuration.newClient(configuration.writeConcernMax(), { maxPoolSize: 1 });
-      client.connect(function (err, client) {
-        var db = client.db(configuration.db);
-        var number_of_tests_done = 0;
-
-        var collection = db.collection('test_object_id_generation.data');
-        // Insert test documents (creates collections and test fetch by query)
-        collection.insert({ name: 'Fred', age: 42 }, { writeConcern: { w: 1 } }, function (err, r) {
+      var db = client.db(configuration.db);
+      var number_of_tests_done = 0;
+
+      var collection = db.collection('test_object_id_generation.data');
+      // Insert test documents (creates collections and test fetch by query)
+      collection.insertMany(
```

Comment:
how come we changed the test to use `insertMany` instead of `insert`?

Reply:
insert is insertMany